features/GettingStarted.md

```diff
@@ -2,44 +2,62 @@
 
 Install Rails 6
 
+```shell
     $ gem install rails -v "~> 6.0.0"
+```
 
 ### Generate an app
 
+```shell
     $ rails new example_app
     $ cd example_app
+```
 
 ### Add `rspec-rails` to the Gemfile
 
+```shell
```

Comment:
There’s a slight [difference](https://stackoverflow.com/a/52586193) between how console and sh would render. But I have no certainty that it’s supported by Github-flavoured markdown and what we use in their  docs (highlight.js?) similarly.

Reply:
It looks like it supports the designation as console in that it generates a <pre> block with class "highlight console" instead of just keeping ```console as part of an unrecognized code block. Would you prefer I use console instead of shell? That's an easy change. 

Also, while I'm touching that, would you prefer the page with "$" at the start of console commands be normalized to remove them? That's the only page I've seen it on in your documentation.

Reply:
In the updated version I just pushed, I have updated all use of shell to console.